app/presenters/queries/expanded_link_set.rb

```diff
@@ -3,10 +3,11 @@ module Queries
     class ExpandedLinkSet
       attr_reader :state_fallback_order
 
-      def initialize(content_id:, state_fallback_order:, locale_fallback_order: Edition::DEFAULT_LOCALE)
+      def initialize(content_id:, state_fallback_order:, locale_fallback_order: Edition::DEFAULT_LOCALE, edition_id: nil)
```

Comment:
Although it is unnecessary coupling, we believe there would be performance issues with performing the query to find the `edition_id` when it is already available in the places this class is used.